layerfetcher/wire_types.go

```diff
@@ -2,7 +2,17 @@ package layerfetcher
 
 import "github.com/spacemeshos/go-spacemesh/common/types"
 
-// layerBlocks is the response for layer hash
+// layerHash is the response for a given layer
```

Comment:
it seems like a copy from another struct. better to remove it altogether

Reply:
done